### PR TITLE
Remove the unnecessary type conversions

### PR DESCRIPTION
**Pull request by lee-lei**
## What changes were proposed in this pull request?

1. Use concise scala style to remove unnecessary conversion.

## How was this patch tested?
run unit tests and all of them passed.

### CODE REVIEW

**Comment by lee-lei** [comment]
@adrian-wang @linhong-intel @yueawang @chenghao-intel Please help review it when you are available. Thanks

File: src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapScanner.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ import scala.collection.JavaConverters._
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FSDataInputStream, Path}
 import org.roaringbitmap.buffer.ImmutableRoaringBitmap
-import org.roaringbitmap.buffer.MutableRoaringBitmap
+import org.roaringbitmap.buffer.BufferFastAggregation
```
Comment (linhong-intel) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapScanner.scala]: import order.
Intellij has style checker. Make sure you checked all Intellij style warnings before upload.

File: src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapScanner.scala [hunk 2]
```diff
@@ -120,22 +120,18 @@ private[oap] case class BitMapScanner(idxMeta: IndexMeta) extends IndexScanner(i
     bmUniqueKeyListBuffer
   }
 
-  private def readBmUniqueKeyListFromCache(cr: CacheResult): mutable.ListBuffer[InternalRow] = {
+  private def readBmUniqueKeyListFromCache(cr: CacheResult): immutable.IndexedSeq[InternalRow] = {
```
Comment (linhong-intel) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapScanner.scala]: immutable is unnecessary.

Comment (chenghao-intel) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapScanner.scala]: Should be `IndexedSeq` or simply a `Seq`? And `immutable.` can be removed as it's imported by default.

Comment (lee-lei) [reply src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/BitMapScanner.scala]: fixed. IndexedSeq is the default structure expected by compiler. Thanks.

**Comment by linhong-intel** [comment]
good to me. no conflict with my code.

**Comment by adrian-wang** [comment]
LGTM

**Pull request merged**